File: src/main/java/pl/uwr/server/model/CancellationRequest.java
package pl.uwr.server.model;

import lombok.Data;
import lombok.NoArgsConstructor;

@NoArgsConstructor
public class CancellationRequest {
    private Credentials requester;
    private Long classId;

    public CancellationRequest(Credentials requester, Long classId) {
        this.requester = requester;
        this.classId = classId;
    }

    public Credentials getRequester() {
        return requester;
    }

    public void setRequester(Credentials requester) {
        this.requester = requester;
    }

    public Long getClassId() {
        return classId;
    }

    public void setClassId(Long classId) {
        this.classId = classId;
    }
}

File: src/main/java/pl/uwr/server/model/CancellationRequest.kt
package pl.uwr.server.model

import lombok.NoArgsConstructor

@NoArgsConstructor
/**
 * klasa reprezentująca żądanie anulowania rezerwacji przez użytkownika
 * @param requester dane użytkownika anulującego rezerwację
 * @param reservationId ID rezerwacji
 */
class CancellationRequest(var requester: Credentials, var reservationId: Long)